test_runner/build.gradle.kts

```diff
@@ -132,12 +132,18 @@ publishing {
     }
 }
 
+subprojects {
```

Comment:
`test_runner` doesn't have any subprojects (or am I wrong?) so `subprojects` is redundant here.
But I think we could move it to the root build gradle file

Reply:
Ah... I think `test_runner:test` task can't use parallelism. We have failed tests then.
Anyway, IMHO it would be worth to have global config with parallel forks and explicit single fork configuration for `test_runner` project (with comment why). We can also create an issue to make it possible to test in parallel

Reply:
Hey Pawel it was here before, it is just moved upper.
If we move it to root `build.gradle.kts` then it makes that tests fail as you mention, so we could remove it from here

Reply:
Sounds good to me 👍

Reply:
lets remove it for now and think on it later. It can greatly speedup the build but it has some issues like the failed tests

Reply:
Are you sure? I think it has no effect if placed here

Reply:
I edited my comment ;)